src/presets.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   defaults: {
-    System: ['OS', 'CPU', 'Memory', 'Container', 'Shell'],
+    System: ['OS', 'CPU', 'Memory', 'Container', 'Shell', 'GLibc'],
```

Comment:
Not sure I like this one in System. Does it make more sense to put it somewhere else?

Reply:
https://en.m.wikipedia.org/wiki/GNU_C_Library, I think most near place is system.

Reply:
I still feel like GLibc is more akin to Clang, make and gcc, rather than CPU or memory.

Reply:
My first intention on implementing this is that Node.js issue report require this. But I do think it's a core part for unix-like system, so I still want to put to `System` part.

Reply:
ping

Reply:
@tabrindle ping

Reply:
I bring `envinfo` to `Node.js` github action pipeline, see https://github.com/nodejs/node/blob/f0a33d99ec4656b87bebb97162837047b52eb6f8/.github/workflows/CI.yml#L20. Also want to bring it to issue report. If glibc not in this location, will be not intuitive in my opinion.

Reply:
I agree with @tabrindle on this one.  Put it in `utilities` and we can then maybe break utilities out to more semantic sections (like c-related utilities, git, etc).

Reply:
Can we comprimise on creating a new system level software category? 

This doesn't belong with CPU/memory, but can see how it would belong with shell, make and gcc.

Reply:
I put it in `utilility` category.

Reply:
Perhaps CPU/Memory etc should be under hardware, and System should contain OS, container, shell, glibc etc.